cloudsim-plus-testbeds/src/main/resources/AwsEC2Prices

```diff
@@ -0,0 +1,48 @@
+  {
```

Comment:
You must have one json file for each different configuration.
The file should be named with the instance name, such as t2.nano.json (always include the extension)
The content of each file should be like that:
{
         "instanceName": "t2.nano",
         "vCPU":1,
         "memoryInMB":512,
         "pricePerHour":0.0082
       }